scripts/superdesk-search/search.js

```diff
@@ -1586,6 +1586,14 @@
                             scope.searchProviderTypes = searchProviderService.getProviderTypes();
                             scope.cvs = metadata.search_cvs;
                             scope.search_config = metadata.search_config;
+                            scope.scanpix_subscriptions = [{
+                                name: 'subscription',
+                                label: 'inside subscription',
+                            }, {
+                                name: 'all',
+                                label: 'all photos',
```

Comment:
translation using `gettext` would be better here, because from here it can extract those strings.